packages/core/src/textures/resources/autoDetectResource.ts

```diff
@@ -57,7 +59,7 @@ export const INSTALLED = [];
  *        texture should be updated from the video. Leave at 0 to update at every render
  * @return {PIXI.resources.Resource} The created resource.
  */
-export function autoDetectResource(source, options)
+export function autoDetectResource(source: any, options: any): Resource
```

Comment:
The options here are not any

Reply:
they are being passed to autoDetectResource, we only can do something like `{[key: string: any}`

Reply:
Yes we do. The options are in the documentation.

Reply:
We can add those fields to satisfy autocomplete but we need `{[key:string]:any} just because its possible to use in custom resources.

Reply:
We should define strict types and then add a generic type for extensibility. Just using the generic type isn’t that great

Reply:
I dont care about this place and it can be patched later when we find a case it doesnt work. Its still experimental API :) 

You may change it and and other related places, and commit to this branch.